goreleaser.example.yml

```diff
@@ -0,0 +1,81 @@
+# Build customization
+build:
+  # main.go path.
+  # Default is `main.go`
+  main: ./cmd/main.go
+
+  # name of the binary. Default is the name of the project directory.
+  binary_name: program
+
+  # custom ldflags.
+  # Default is `-s -w`
+  ldflags: -s -w
+
+  # GOOS list to build in.
+  # Defaults are darwin and linux
+  goos:
+    - freebsd
+    - windows
+
+  # GOARCH to build in.
+  # Defaults are 386 and amd64
+  goarch:
+    - amd64
+
+
+# Archive customization
+archive:
+  # You can change the name of the archive.
+  # This is parsed with golang template engine and the following variables
+  # are available:
+  # - BinaryName
+  # - Version
+  # - Os
+  # - Arch
+  # The default is `{{.BinaryName}}_{{.Os}}_{{.Arch}}`
+  name_template: "{{.BinaryName}}_{{.Version}}_{{.Os}}_{{.Arch}}"
+
+  # Archive format. Valid options are `tar.gz` and `zip`.
+  # Default is `zip`
+  format: zip
+
+  # Replacements for GOOS and GOARCH on the archive name.
+  # The left values should be a valid GOOS or GOARCH name, and the right
```

Comment:
Maybe: "The keys should be valid GOOS or GOARCH values followed by your custom replacements."